src/get-aws-results.ts

```diff
@@ -72,10 +72,22 @@ export const getAwsResults = async (
        * no idea what service is being requested. You're just lucky this isn't the v3 AWS SDK
        * where there's a chance we're going to end up doing an eval so we still be service agnostic
        */
-      const client = new (<any>AWS)[service](clientParams);
+      let client;
+      try {
+        client = new (<any>AWS)[service](clientParams);
+      } catch (error) {
+        console.error(`Unable to instantiate service: ${service}`);
```

Comment:
I think we need to `throw` these. I had a whole thing about when and where we output to console and javascript best practices, and in this object constructor in a shared library it feels like major bad juju to be dumping errors to console rather than throwing them as exceptions.

Reply:
OK - will do that.

Reply:
👍